addons/account_edi_ubl_cii/tests/test_ubl_cii.py

```diff
@@ -56,3 +57,24 @@ def test_import_product(self):
         )
 
         self.assertRecordValues(new_invoice.invoice_line_ids, line_vals)
+
+    def test_get_norway_partner_party_vals_with_no_tax(self):
+        partner = self.env["res.partner"].create({
+            "name": "Test partner",
+            "country_id": self.env.ref('base.no').id,
+        })
+
+        partner_party_vals = self.env["account.edi.xml.ubl_bis3"]._get_partner_party_vals(partner, role='customer')
+        new_invoice = self.env["account.move"].create({
+            "partner_id": partner.id,
+            "move_type": "out_invoice",
+            "invoice_line_ids": [Command.create({"name": "Test product", "price_unit": 100})],
+        })
+        new_invoice.action_post()
+        xml_content = self.env['account.edi.xml.ubl_bis3']._export_invoice(new_invoice)
+        root = etree.fromstring(xml_content[0])
+        xpath_to_endpoint = './{*}AccountingCustomerParty/{*}Party/{*}EndpointID'
+        self.assertEqual(partner_party_vals['endpoint_id'], partner.vat)
+        self.assertEqual(root.findtext(xpath_to_endpoint), partner.vat or None)
```

Comment:
Just a few cosmetic changes before I r+ :) : 
```suggestion
    def test_norway_partner_without_vat(self):
        partner = self.env["res.partner"].create({
            "name": "Norwegian partner",
            "country_id": self.env.ref('base.no').id,
        })

        new_invoice = self.env["account.move"].create({
            "partner_id": partner.id,
            "move_type": "out_invoice",
            "invoice_line_ids": [Command.create({"name": "Test product", "price_unit": 100})],
        })
        new_invoice.action_post()
        xml = self.env['account.edi.xml.ubl_bis3']._export_invoice(new_invoice)[0]
        root = etree.fromstring(xml)
        self.assertEqual(root.find('./{*}AccountingCustomerParty/{*}Party/{*}EndpointID'), None)
```
Also do not forget to leave one line after this test (you currently have 2 :) )